packages/service-library/src/servicelib/aiohttp/long_running_tasks/_dependencies.py

```diff
@@ -1,8 +1,21 @@
+from typing import Any
+
 from aiohttp import web
 
 from ...long_running_tasks._task import TasksManager
-from ._constants import APP_LONG_RUNNING_TASKS_MANAGER_KEY
+from ._constants import (
+    APP_LONG_RUNNING_TASKS_MANAGER_KEY,
+    RQT_LONG_RUNNING_TASKS_CONTEXT_KEY,
+)
 
 
 def get_tasks_manager(app: web.Application) -> TasksManager:
     return app[APP_LONG_RUNNING_TASKS_MANAGER_KEY]
+
+
+def get_task_context(request: web.Request) -> dict[str, Any]:
+    return request[RQT_LONG_RUNNING_TASKS_CONTEXT_KEY]
+
+
+def create_task_name_from_request(request: web.Request) -> str:
+    return f"{request.method} {request.rel_url}"
```

Comment:
STYLE: add some prefix that identifies it as long-running task. This might help to filter ?

Reply:
this is returned by ```GET /tasks``` I guess the client knows it is a task no?

Reply:
besides I will need to sync with @odeimaiz to see what he wants to retrieve the tasks initiated by the frontend.